extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +119,40 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
```

Comment:
Please inline above where you assign the promise. Things like this are the best in the build or layout callback, especially since the `playerReadyResolver_ ` is declared right here. It's a bit verbose, but it's ok.